astroid/nodes/node_classes.py

```diff
@@ -779,8 +797,21 @@ def fromlineno(self) -> int:
 
     @cached_property
     def arguments(self):
-        """Get all the arguments for this node, including positional only and positional and keyword"""
-        return list(itertools.chain((self.posonlyargs or ()), self.args or ()))
+        """Get all the arguments for this node. This includes:
+        * Positional only arguments
+        * Positional arguments
+        * Keyword arguments
+        * Variable arguments (.e.g *args)
+        * Keyword only arguments (e.g **kwargs)
```

Comment:
Keyword-only arguments are ones that follow `*`, so to describe `**kwargs` we need to say something like "variable keyword arguments"